pkg/controller/util/node/controller_utils.go

```diff
@@ -122,31 +125,38 @@ func MarkAllPodsNotReady(kubeClient clientset.Interface, node *v1.Node) error {
 	nodeName := node.Name
 	klog.V(2).Infof("Update ready status of pods on node [%v]", nodeName)
 	opts := metav1.ListOptions{FieldSelector: fields.OneTermEqualSelector(api.PodHostField, nodeName).String()}
-	pods, err := kubeClient.CoreV1().Pods(metav1.NamespaceAll).List(opts)
-	if err != nil {
-		return err
-	}
-
+	podPager := pager.New(pager.SimplePageFunc(func(opts metav1.ListOptions) (runtime.Object, error) {
+		return kubeClient.CoreV1().Pods(metav1.NamespaceAll).List(opts)
+	}))
 	errMsg := []string{}
-	for _, pod := range pods.Items {
+	err := podPager.EachListItem(context.Background(), opts, func(obj runtime.Object) error {
```

Comment:
Given https://golang.org/pkg/context/#Background I believe this should probably be context.TODO() as this is not the top level code wiring in the context.

Reply:
Yes, you're right. That's an interesting distinction. I'll switch.